updater/updater.go

```diff
@@ -44,6 +47,19 @@ var (
 	unsafeHTTP      = flag.Bool("unsafe", false, "Debug Only: Support non-https update checks for testing.")
 )
 
+const (
+	profileFileName   string = "updater-profile.conf"
+	keyIdentity       string = "identity"
+	keyChannel        string = "channel"
+	valChannelStable  string = "stable"
+	valChannelBeta    string = "beta"
+	valChannelExp     string = "experimental"
+	customHeader      string = "X-profile-"
+	idHeaderStr       string = customHeader + keyIdentity
+	channelHeaderStr  string = customHeader + keyChannel
+	timezoneHeaderStr string = customHeader + "timezone"
+)
```

Comment:
customHeader var is not much reusable. 
it would be sometime okay to repeat same prefix in the vars so that it would be more readable  and also, variable names are not quite understandable.

for example.

```
headerProfileIDKey       = "X-profile-identity"
headerProfileChannelKey  = "X-profile-channel"
headerProfileTimezoneKey = "X-profile-timezone"
```
The keys indicate that they are used for profile as I don't know much about it but at least I can guess.
and get rid of all constants with string type which wouldn't be needed in this case.
what do you think?

Reply:
Will change